### PR TITLE
Add endpoints for pocket proxy #121

### PR DESCRIPTION
**Pull request by jaredkerim**

### CODE REVIEW

**Comment by jaredkerim** [comment]
@oyiptong @ncloudioj r?

File: embedly/embedly/pocket.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+            except Exception:
+                raise self.PocketException(
+                    'Unable to start the pocket fetch job.')
+                statsd_client.incr('request_recommended_job_create_fail')
+
+            statsd_client.incr('request_recommended_job_create')
+
+            self.redis_client.set(
+                self.redis_key, self.redis_in_flight_value)
+            self.redis_client.expire(self.redis_key, self.job_ttl)
```
Comment (ncloudioj) [inline embedly/embedly/pocket.py]: Oops, too many empty lines here.

File: embedly/embedly/api/views.py [hunk 2]
```diff
@@ -46,36 +55,52 @@ def extract_urls_v2():
...
+
+@blueprint.route('/v2/recommendations', methods=['GET'])
+def get_recommended_urls():
+    response_data = {
+        'urls': {},
+        'error': '',
+    }
+
+    try:
+        response_data['urls'] = (
```
Comment (ncloudioj) [inline embedly/embedly/api/views.py]: nit: redundant brackets?

File: embedly/embedly/pocket.py [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+class PocketClient(object):
+
+    class PocketException(Exception):
+        pass
+
+    def __init__(self, pocket_key, redis_client, redis_data_timeout,
+                 job_queue, job_ttl):
+        self.pocket_url = (
+            'https://getpocket.com/v3/firefox/'
```
Comment (ncloudioj) [inline embedly/embedly/pocket.py]: Better take this to the `get_config()`

File: embedly/embedly/pocket.py [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+        for recommended_url in pocket_data['list']:
+            recommended_urls.append({
+              'url': recommended_url['dedupe_url'],
+              'timestamp': ((
+                  int(recommended_url['published_timestamp']) or
+                  int(time.time())
+              ) * 1000),
+            })
+
+        self.redis_client.set(self.redis_key, json.dumps(recommended_urls))
```
Comment (ncloudioj) [inline embedly/embedly/pocket.py]: Handle the Redis exceptions?

File: embedly/embedly/pocket.py [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+        self.redis_client.expire(self.redis_key, self.redis_data_timeout)
```
Comment (ncloudioj) [inline embedly/embedly/pocket.py]: I think Redis.setex() does the above two ops (set with a ttl) for you.

**Comment by ncloudioj** [comment]
LGTM, r+ after fixing the nits.

File: embedly/embedly/pocket.py (hunk 1 above)
Comment (jaredkerim) [inline embedly/embedly/pocket.py]: I like to cluster lines by semantic grouping with empty lines in between, helps me see clusters of logic better.

File: embedly/embedly/api/views.py (hunk 2 above)
Comment (jaredkerim) [inline embedly/embedly/api/views.py]: This is how you split lines in Python.

File: embedly/embedly/pocket.py (hunk 3 above)
Comment (jaredkerim) [inline embedly/embedly/pocket.py]: Good call.

File: embedly/embedly/pocket.py (hunk 4 above)
Comment (jaredkerim) [inline embedly/embedly/pocket.py]: Yes good idea.

File: embedly/embedly/pocket.py (hunk 5 above)
Comment (jaredkerim) [inline embedly/embedly/pocket.py]: Oh good call!

File: embedly/embedly/pocket.py (hunk 1 above)
Comment (jaredkerim) [inline embedly/embedly/pocket.py]: Oops!

File: embedly/embedly/pocket.py (hunk 5 above)
Comment (jaredkerim) [inline embedly/embedly/pocket.py]: Gonna do this in #126

**Pull request merged**